content/en/getting_started/synthetics/private_location.md

```diff
@@ -42,7 +42,10 @@ Your private locations test results display identically to your managed location
 
 ## Create your private location
 
-1. Install [Docker][4] on a machine. To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].
+1. You can use any Unix virtual machine with Docker or without a virtual machine. Install [Docker][4] on a machine, or use another container runtime such as [Podman][10].
+
+   To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].
```

Comment:
I may have read too fast, though I think the `[2]` link points to the Synthetics PL docker image and not to an Ubuntu 16.04 vagrant box. That would be `https://app.vagrantup.com/ubuntu/boxes/xenial64`

Also, the version 16.04 had its support end in April 2021, could you suggest using 22.04 (`https://app.vagrantup.com/ubuntu/boxes/jammy64`)?